spec/cypress/app_commands/scenarios/admin/schools.rb

```diff
@@ -3,7 +3,9 @@
 school = FactoryBot.create(:school, name: "Include this school", urn: 123_456)
 local_authority = FactoryBot.create(:local_authority)
 SchoolLocalAuthority.create!(school: school, local_authority: local_authority, start_year: 2021)
-FactoryBot.create(:user, :induction_coordinator, full_name: "Sarah Smith", email: "sarah.smith@example.com", schools: [school])
```

Comment:
This was an interesting one - creating a user with an induction_coordinator profile would create a school which had a constant name but a random URN.